Adding pyhit as a conda package

milljm opened the pull request:
Closes #15582

Commit:
Adding pyhit as a conda package

Closes #15582

moosebuild commented:
All [jobs](https://civet.inl.gov/event/56825/) on 02f37f7 : invalidated by @milljm

fixed upstream moose-pyhit Makefile.

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/562669/) on 02f37f7 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/15583/site)

This comment will be updated on new commits.

lindsayad commented on conda/pyhit/meta.yaml:
```diff
@@ -0,0 +1,34 @@
+{% set build = 0 %}
+{% set version = "2020.09.10" %}
```
The version is in the future?

milljm replied:
Uggh... that sucks...

Commit:
Correct the data/version

correcting the date/version in meta file. Behind the scenes, I also now have to
physically remove the pr channel on rod. As that package will always be 'newer'
according to Conda.

Closes #15582
